src/Uno.UI/UI/Xaml/FrameworkElement.Interface.wasm.cs

```diff
@@ -220,14 +243,23 @@ public bool IsEnabled
 					options: FrameworkPropertyMetadataOptions.Inherits,
 					propertyChangedCallback: (s, e) =>
 					{
+						Console.WriteLine($"{((FrameworkElement)s).GetHashCode():X8} property changed IsEnabled: {e.NewValue}");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5714227)